### PR TITLE
Improve clang-format

### PR DESCRIPTION
**Pull request by gleon99**

### CODE REVIEW

**Comment by gleon99** [comment]
@yosefe - that's your original PR + my commit of "CR fixes".

File: ucx/.clang-format [hunk 1]
```diff
@@ -52,7 +52,7 @@ BraceWrapping:
 BreakBeforeBinaryOperators: false
 BreakBeforeTernaryOperators: false
 BreakStringLiterals: true
-ContinuationIndentWidth: 8
+ContinuationIndentWidth: 4
```
Comment (yosefe) [inline ucx/.clang-format]: why? IMO 8 is better

File: ucx/.clang-format [hunk 2]
```diff
@@ -21,9 +21,9 @@ AllowShortLoopsOnASingleLine: false
 AllowShortEnumsOnASingleLine: false
 AllowDesignatedInitializersOnASingleLine: false
 AlwaysBreakAfterReturnType: None
-PenaltyReturnTypeOnItsOwnLine: 5
-PenaltyBreakAssignment: 200
-PenaltyExcessCharacter: 200
+PenaltyReturnTypeOnItsOwnLine: 100
+PenaltyBreakAssignment: 100
+PenaltyExcessCharacter: 10
```
Comment (yosefe) [inline ucx/.clang-format]: low penalty here means more likely to keep lines longer, IMO it should be higher

Comment (gleon99) [reply ucx/.clang-format]: Fixed

File: ucx/.clang-format (hunk 1 above)
Comment (gleon99) [reply ucx/.clang-format]: Fixed

**Review by yosefe — APPROVED** [review]

**Pull request merged**